lib/cli/generators/GATSBY/index.js

```diff
@@ -0,0 +1,55 @@
+import mergeDirs from 'merge-dirs';
+import path from 'path';
+import fs from 'fs';
+import { getVersions, getPackageJson, writePackageJson } from '../../lib/helpers';
+
+export default async npmOptions => {
+  const [
+    storybookVersion,
+    actionsVersion,
+    linksVersion,
+    addonsVersion,
+    babelCoreVersion,
+    babelRuntimeVersion,
+  ] = await getVersions(
+    npmOptions,
+    '@storybook/react',
+    '@storybook/addon-actions',
+    '@storybook/addon-links',
+    '@storybook/addons',
+    'babel-core',
```

Comment:
we should either install `babel-core@bridge` + `@babel/core` here (see `installBabel` helper), or add a babel config, because our default config is now incompatible with Babel 6